docs/import.md

```diff
@@ -0,0 +1,113 @@
+# Importing vim menus to V̂enu
+
+V̂enu can import vim's native menu entries (`:help menu`). This document
+provides reference for `vim#native` module used for this purpose.
+
+## Functions
+
+---
+
+```vim
+venu#native#import([menu_filter])
+```
+
+Imports VIM native menu entries to V̂enu.
+
+Optionally takes a callback function `menu_filter` used for filtering which
+entries to import and modifying menu entries before they get imported. By
+default, the value of `g:venu_native_menu_filter` is used if provided,
+otherwise `venu#native#defaultFilter` is used. See [Filtering Menus](#filtering-menus)
+section for more details about menu filters.
+
+---
+
+```vim
+venu#native#parseMenu([menu_cmd])
+```
+
+Parses output of vim menu printing command and returns a list of menu
+dictionaries.
+
+Optionally takes menu command `menu_cmd` to use for printing vim menus. By
+default, the value of `g:venu_native_menu_cmd` is used if provided, otherwise
+`amenu` is used.
+
+Note that returned menu dictionaries have different format than those used by
+V̂enu. Menu dictionaries returned by this function have these keys:
+
+- `name`: Name of the menu entry. It doesn't include right aligned text
+(provided by `desc`) or ampersand preceding the shortcut character (provided
+by `shortcut`).
+- `desc`: Right aligned text of the menu entry, if there is one. This is the
+portion following `<TAB>` special character in the menu entry name.
+- `priority`: Priority of the menu entry. Menus with a higher priority go more
+to the right.
```

Comment:
What does "more to the right" mean? Isn't the priority the number? E.g.
```
75 Riv
  500 Project
    500 Index^I<C-E>ww
```
Riv has priority 75, Project has priority 500, etc?

Reply:
If two menus are in the same level, the one with higher priority goes to right in vim's menu. I quoted that sentence directly from vim's `:help menu` but I guess I could word it better. How about this: `If two menus are in the same level, the one with higher priority value will be closer to the right side.`